web/web.go

```diff
@@ -391,7 +391,21 @@ func (h *Handler) Run(ctx context.Context) error {
 	)
 	av2.RegisterGRPC(grpcSrv)
 
-	hh, err := av2.HTTPHandler(grpcl.Addr().String())
+	lAddr := grpcl.Addr().String()
+	host, port, err := net.SplitHostPort(lAddr)
+	if err != nil {
+		return err
+	}
+	ip := net.ParseIP(host)
+	if ip.Equal(net.IPv4zero) || ip.Equal(net.IPv6zero) {
+		// The gRPC server is listening on an empty/wildcard address (0.0.0.0 or ::).
+		// We should be able to connect to an empty IP just fine, but as there were
```

Comment:
This is a server, why does it matter how clients handle addresses?

Reply:
because the REST gateway starts a client to the gRPC server in order to proxy the HTTP requests.